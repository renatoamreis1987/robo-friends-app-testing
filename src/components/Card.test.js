import { shallow } from 'enzyme';
import React from 'react'
import Card from './Card';

// shallow is the one we will use 90% of the time

// mount 

// render - It's use to render react components to a static html


it('expect to render Card Component', () => {
    expect(shallow(<Card />)).toMatchSnapshot()
})
