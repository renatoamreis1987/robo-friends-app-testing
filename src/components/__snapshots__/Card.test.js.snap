// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card Component 1`] = `ShallowWrapper {}`;
